docs/reference/cat/nodes.asciidoc

```diff
@@ -285,17 +285,14 @@ Number of suggest operations, such as `0`.
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=help]
 
-include::{docdir}/rest-api/common-parms.asciidoc[tag=local]
-+
---
 `local`::
-(Optional, boolean) If `true`, the request computes the list of selected nodes
-from the local cluster state. Defaults to `false`, which means the list of
-selected nodes is computed from the cluster state on the master node. In either
-case the coordinating node sends a request for further information to each
-selected node. deprecated::[7.6,This parameter does not cause this API to act
-locally. It will be removed in version 8.0.]
---
+(Optional, boolean) 
+deprecated:[7.6.0,This parameter does not cause this API to act locally. It will be removed in version 8.0.]
+If `true`, the request computes the list of selected nodes from the local
+cluster state. Defaults to `false`, which means the list of selected nodes is
+computed from the cluster state on the master node. In either case the
```

Comment:
While we're at it: needs a `,` after "In either case" doesn't it? :)

Reply:
No, I think it's acceptable (and better) as it is.